styleguide/_themes/global/scss/fonts.scss

```diff
@@ -20,7 +21,7 @@ $code: Tahoma, Arial, Geneva, Helvetica;
 }
 
 @mixin copy($color: $gray-darkest) {
-  color: $gray-darkest;
+  color: $color;
```

Comment:
I vaguely remember there being a reason for this, but can't recall what it is now.